{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/config.py

```diff
@@ -113,6 +113,20 @@ def _(x):
 {%- elif cookiecutter.database == 'mysql'%}
 SQLALCHEMY_DATABASE_URI = \
     'mysql+pymysql://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@localhost/{{cookiecutter.project_shortname}}'
+
+# Disable Flask-DebugToolbar SQLAlchemy pane
```

Comment:
I would disable everything here, not only the SQLAlchemy one, it will be more obvious for the developer that the toolbar is not working as a whole (it is less obvious that only the sqlalchemy part is missing) and it will point them to look at the config.
Also please add into the comment here something like `Disable Flask-DebugToolbar only for mysql database configuration due to the issue ` so it is clear that this was added to the config because of the database type choice